src/pages-helpers/pupil/options-pages/getAvailableProgrammeFactor.ts

```diff
@@ -0,0 +1,62 @@
+import { groupBy, isEqual, uniqWith } from "lodash";
+
```

Comment:
this was previously with the View but it is first used by options-pages-helpers.